internal/handlers/pg/fetch.go

```diff
@@ -73,3 +73,37 @@ func nextRow(rows pgx.Rows) (*types.Document, error) {
 
 	return doc.(*types.Document), nil
 }
+
+func (h *Handler) protoFetch(ctx context.Context, param fetchParam) ([]*types.Document, error) {
+	var sql string
+	if param.comment != "" {
+		param.comment = fmt.Sprintf("/* %s */", param.comment)
+		sql = fmt.Sprintf(`SELECT %s _jsonb FROM %s`, param.comment, pgx.Identifier{param.db, param.collection}.Sanitize())
```

Comment:
why not to put the row above inline in the `Sprintf`?

Reply:
it is a good suggestion :)
but in this case I'm trying to avoid long lines and just explicitly expressed the operation (line 80)

Reply:
yeah it's more visible though